examples/mnist.py

```diff
@@ -97,10 +97,11 @@ def create_iter_functions(dataset, output_layer,
     batch_slice = slice(
         batch_index * batch_size, (batch_index + 1) * batch_size)
 
-    objective = lasagne.objectives.Objective(output_layer, loss_function=lasagne.objectives.multinomial_nll)
+    objective = lasagne.objectives.Objective(output_layer,
+        loss_function=lasagne.objectives.categorical_crossentropy)
 
     loss_train = objective.get_loss(X_batch, target=y_batch)
-    loss_eval = objective.get_loss(X_batch, target=y_batch, deterministic=True)
+    loss_eval = objective.get_loss(X_batch, target=y_batch,deterministic=True)
```

Comment:
Do they need to be explicitly included in a config file somewhere? It would be great if all *.py files were checked.